starter/src/main/java/com/vivy/springdoc/OpenAPIAutoConfiguration.java

```diff
@@ -32,4 +50,66 @@ public OpenAPI bearerOpenAPI() {
                 ))
                 .addSecurityItem(new SecurityRequirement().addList("bearer-jwt", List.of("read", "write")));
     }
+
+    @Bean
+    @ConditionalOnBean(OpenAPI.class)
+    public ApplicationRunner applicationRunner(Environment environment) {
```

Comment:
Dear reviewer, one open issue is how to run this only in test scope. Pls suggest something.